reader/components/main-area/MainArea.scss

```diff
@@ -38,7 +38,9 @@ $sidebar-width: 20rem;
   transition: left 200ms ease;
   will-change: left;
 
-  .sidebar-open & {
-    left: $sidebar-width;
+  @include media-breakpoint-up(md) {
+    .sidebar-open & {
+      left: $sidebar-width;
```

Comment:
I feel like this is not related to the links? This is the sidebar keeping in the screen above `md`. Is that correct?

Reply:
Yes, it is. Maybe I could have created separate PR.